### DISCUSSION TITLE
如何利用 ollama 的web api 去调用 function call  有没有具体报文结构?

### DISCUSSION THREAD

**Discussion by anan1213095357** [QwenLM/Qwen3#987] [General]
有人懂吗 具体结构截个图我看看就行了

**Comment by jklj077** [comment]
这里有ollama python sdk的https://qwen.readthedocs.io/zh-cn/latest/framework/function_call.html#ollama

**Comment by thiner** [comment]
如果你是问API的调用格式，Ollama官方文档就有：https://github.com/ollama/ollama/blob/main/docs/api.md#chat-request-with-tools
